### ISSUE TITLE
Calculating standardized variance using FindVariableFeatures()

### ISSUE THREAD

**Issue by atchen** [satijalab/seurat#3946]
I had a question about how the standardized variance is calculated using the FindVariableFeatures() function, specifically for the 'vst' selection method. 

I read the [methods](https://www.cell.com/cell/fulltext/S0092-8674(19)30559-8), and I understand how the standardized variance is calculated for each feature for each cell (z_ij). However, since this is for each cell, are all the standardized variances for a specific gene averaged, and then the resulting value is reported as the standardized variance for that gene? Or is it the maximum standardized variance that gets reported as that value for the gene?

Basically, I'm confused on how the standardized variance per gene per cell gets translated into the standardized variance for a gene. Thanks for your help!

**Comment by saketkc** [comment]
z_ij is the standardized variance for feature i in cell j.  Variances are then calculate on z_ij for all j for a given gene i.
